DQM/L1TMonitor/python/L1TStage2uGT_cfi.py

```diff
@@ -1,10 +1,11 @@
 import FWCore.ParameterSet.Config as cms
-
 from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
 l1tStage2uGT = DQMEDAnalyzer('L1TStage2uGT',
     l1tStage2uGtSource = cms.InputTag("gtStage2Digis"),    
     monitorDir = cms.untracked.string("L1T/L1TStage2uGT"),
     verbose = cms.untracked.bool(False),
     firstBXInTrainAlgo = cms.untracked.string("L1_FirstCollisionInTrain"),
```

Comment:
Please add the config parameters firstBXInTrainAlgo, lastBXInTrainAlgo, unprescaledAlgoShortList, and prescaledAlgoShortList also to the emulator cfi file. https://github.com/cms-sw/cmssw/blob/master/DQM/L1TMonitor/python/L1TStage2uGTEmul_cfi.py

Reply:
Hi @dtsitson 
This addition to L1TStage2uGTEmul_cfi.py is needed to avoid a crash of the L1TEMU client.